libsel4/include/sel4/bootinfo_types.h

```diff
@@ -24,11 +24,12 @@ enum {
     seL4_CapDomain              = 11, /* global domain controller cap */
     seL4_CapSMMUSIDControl      = 12,  /*global SMMU SID controller cap, null cap if not supported*/
     seL4_CapSMMUCBControl       = 13,  /*global SMMU CB controller cap, null cap if not supported*/
+    seL4_CapSMC                 = 14,  /*global SMC cap, null cap if not supported*/
```

Comment:
It looks like one SMC cap is used for all CPU cores, not sure if this is coarse-grained.

Reply:
That looks fine to me, as long as the intended pattern is that the system initialiser makes badged copies of this cap, handing these out to whoever needs them, restricted to core, sub-system or whichever is needed. Usually the 0-badge version of the cap should stay in the initialiser, or in a high-privilege management component.

That scenario does suggest that we might want more derivation levels for this cap, though, like for endpoint caps.